inc/budget.class.php

```diff
@@ -110,7 +110,6 @@ static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtem
     *     - target for the Form
     *     - withtemplate : template or basic item
     *
-    * @return Nothing (display)
```

Comment:
All methods exepted constructor must hav ea return type documented. `@return void` should be used here.